src/screens/Assets/collectibles/CollectiblesTab.js

```diff
@@ -83,18 +85,38 @@ function CollectiblesTab() {
   const renderItem = (items: CollectibleItem[], chain: Chain) => {
     const itemWidth = (width - 48) / numberOfColumns;
 
+    const dotStyle = styled.View`
+      flex-direction: column,
+      align-items: stretch;
+      padding: 0 ${spacing.mediumLarge}px;
+    `;
+
     return (
-      <ListRow>
-        {items.map((item) => (
-          <CollectibleListItem
-            key={item.key}
-            title={item.title}
-            iconUrl={item.iconUrl}
-            width={itemWidth}
-            onPress={() => navigateToCollectibleDetails(item, chain)}
-          />
-        ))}
-      </ListRow>
+      <>
+        <SwiperFlatList
+          // autoplay
+          // autoplayDelay={5}
+          // autoplayLoop
+          index={0}
+          showPagination={1}
+          data={items}
+          renderAll={1}
```

Comment:
I don't get it, it still renders all? also `renderAll` prop accepts boolean param so just `renderAll` is ok